vermouth/tests/test_molecule.py

```diff
@@ -1229,6 +1229,7 @@ def test_str_method(mol, moltype):
     found = str(mol)
     assert '{} {}'.format(len(mol), 'atoms') in found
     hypothesis.assume('' not in mol.interactions)
+    hypothesis.assume(all(itype not in '0123456789' for itype in mol.interactions))
```

Comment:
Why couldn't we have digits in interaction names?

Reply:
It was because I check `if itype not in found` if there are no interactions associated. And a valid `str(molecule)` *can* contain digits. However, correcting the test opened up a lot of edge cases where you have e.g. the following: `interactions = {'aa': [...], 'a': []}` causing failures.
Test has been adjusted to ensure `0 a` is not in found.